packages/client/lib/service/txpool.ts

```diff
@@ -398,7 +540,7 @@ export class TxPool {
 
     const newTxHashes = []
     for (const tx of txs) {
-      this.add(tx)
+      await this.add(tx)
```

Comment:
this needs a try/catch around it in case the tx is invalid, we don't want to abort processing the other txs

Reply:
Ouch, good catch :open_mouth: :+1: